### DISCUSSION TITLE
WebOs 23 will truncate service menu options - add to documentation a command for full access.

### DISCUSSION THREAD

**Discussion by msky73** [JPersson77/LGTVCompanion#207] [Ideas]
Hi
The latest update of older TVs to WebOS 23 will shorten service menu options when access to OLED TPC/GSR will be unavailable.
To regain the full service menu, one can send an argument by **-settings_other** command.
The argument is **"{\"svcMenuFlag\": false}"**
so full command shoud be **lgtvcli.exe -settings_other "{\"svcMenuFlag\": false}"**
Could you add it to documentation?

Thank you.

**Comment by JPersson77** [comment]
Thanks! I will add it to the documentation. Do you have a screenshot of how the truncated service menu looks?

**Reply by msky73** [reply]
Hi
I reverted to the truncated menu by
**lgtvcli.exe -settings_other "{\"svcMenuFlag\": true}"**    - see photos attached.

But I need your help :-)
I cannot get to the full menu by reverting the flag again .
After 
**lgtvcli.exe -settings_other "{\"svcMenuFlag\": false}"**
the device says: 
**{"Device1":{"info":"Cannot relay luna response."}}**

Do you happen to have any ideas how to solve it?

---------

_edit: it works now, I have waited 10 minutes for another try._


---------
![s1a](https://github.com/JPersson77/LGTVCompanion/assets/138362291/8eb06df4-d7b2-4e10-b690-afab9800316a)
![s2a](https://github.com/JPersson77/LGTVCompanion/assets/138362291/d5f6bf3a-fb22-411a-9717-cba6e213fae5)

**Reply by DeathStalker77** [reply]
How do you input those command line instructions?

Thanks!

--- DS

**Comment by JPersson77** [comment]
Thanks again! Confirmed this works on my LG 42 (Flex) as well

**Comment by DeathStalker77** [comment]
I don't access the Service Menu (at this point), but I do have the autodim (Energy Mgt) turned off in the main menu.  My unit is a 2023 C2.

Thanks!

--- DS

**Comment by JPersson77** [comment]
Hi @DeathStalker77 

You can access the service menu by using the command line. Open a command prompt, CMD, navigate to the install directory, c:/program files/lgtv companion/ and follow the instructions here: [link](https://github.com/JPersson77/LGTVCompanion/blob/master/Docs/Commandline.md#accessing-the-service-menu-lgtv-companion-and-lgtvcli)
